sal_interfaces/Forecast/Forecast_Telemetry.xml

```diff
@@ -0,0 +1,674 @@
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>modelrun_utc</EFDB_Name>
+      <Description>The time that the forecast was run</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>modelrun_updatetime_utc</EFDB_Name>
+      <Description>The time that the model was last updated</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>Forecast</Subsystem>
+    <EFDB_Topic>Forecast_trend_1h</EFDB_Topic>
+    <item>
+      <IDL_Type>double</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast time</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>forecast_time</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast temperature</Description>
+      <Units>deg</Units>
+      <EFDB_Name>temperature</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast temperature spread</Description>
+      <Units>deg</Units>
+      <EFDB_Name>temperature_spread</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast precipitation</Description>
+      <Units>millimeter</Units>
+      <EFDB_Name>precipitation</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast precipitation spread</Description>
+      <Units>millimeter</Units>
+      <EFDB_Name>precipitation_spread</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast windspeed</Description>
+      <Units>meter/second</Units>
+      <EFDB_Name>windspeed</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast windspeed spread</Description>
+      <Units>meter/second</Units>
+      <EFDB_Name>windspeed_spread</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast wind direction</Description>
+      <Units>deg</Units>
+      <EFDB_Name>wind_direction</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast sea level pressure</Description>
+      <Units>meter*pascal</Units>
+      <EFDB_Name>sea_level_pressure</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast relative humidity</Description>
+      <Units>pct</Units>
+      <EFDB_Name>relative_humidity</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast ghi backwards</Description>
+      <Units>watt*hour*meter**2</Units>
```

Comment:
Shouldn't this be `watt/hour/m**2`? Please review this and all other units that have `*` and that should have `/` instead.

Reply:
Unit is Whm-2.

Reply:
So then it should be `watt*hour/m**2` so please correct that and check all other units as well.